test/jdk/security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java

```diff
@@ -27,12 +27,12 @@
  * @summary Interoperability tests with Actalis CA
  * @library /test/lib
  * @build jtreg.SkippedException ValidatePathWithURL CAInterop
- * @run main/othervm -Djava.security.debug=certpath,ocsp
+ * @run main/othervm/manual/manual -Djava.security.debug=certpath,ocsp
```

Comment:
What is the reason why manual is written twice? (Here and in all other places in this file...)

Reply:
good catch! I have fixed it.